src/error/catch.rs

```diff
@@ -87,18 +87,22 @@ where S: Serializer,
 ///
 /// The HTTP response will include a basic error message in the response body.
 impl Catch for DefaultCatch {
-    type Body = &'static str;
+    type Body = String;
```

Comment:
Unfortunately, I think that changing this type is *technically* a breaking change.

I don't know if it matters in practice.

Reply:
We can't serialize a struct to `&'static str`. Is there anything we can do about that?

Reply:
Either we don't care that it is a breaking change (it is very unlikely it will impact anything) or we bump the version and fix the other stuff (avoid `Result<Result<...>, ...>`).

it would also be nice to get rid of `impl_web!` since attribute proc macros are on stable.

Reply:
Can we have few breaking releases in a row? I'd love to see error messages as soon as possible. It's kind of pain for us with an application that already in production :-)

Reply:
Let's merge it then.